insights/parsers/pcs_config.py

```diff
@@ -0,0 +1,243 @@
+"""
+PCSConfig - command ``pcs config``
+==================================
+
+This module provides class ``PCSConfig`` for parsing output of ``pcs config`` command.
+Typical ``/usr/sbin/pcs config`` output looks something like:
```

Comment:
`like:` should be `like::`

There are some other docstring errors below as well.  To see these, follow the instructions here to generate the docs and review the output: https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring